supervision/dataset/formats/pascal_voc.py

```diff
@@ -17,6 +17,8 @@ def object_to_pascal_voc(
     object_name = SubElement(root, "name")
     object_name.text = name
 
+    xyxy += 1
```

Comment:
Now to keep consistency I'd say we need to add `polygon +=1` in line `33`

Reply:
Added